rinja_parser/src/expr.rs

```diff
@@ -1,18 +1,19 @@
 use std::collections::HashSet;
 use std::str;
 
-use nom::branch::alt;
-use nom::bytes::complete::{tag, take_till};
-use nom::character::complete::{char, digit1};
-use nom::combinator::{consumed, cut, fail, map, not, opt, peek, recognize, value};
-use nom::error::ErrorKind;
-use nom::error_position;
-use nom::multi::{fold_many0, many0, separated_list0, separated_list1};
-use nom::sequence::{pair, preceded, terminated, tuple};
+use winnow::branch::alt;
+use winnow::bytes::{one_of, tag, take_till0};
+use winnow::character::digit1;
+use winnow::combinator::{cut_err, fail, not, opt, peek};
+use winnow::error::ErrorKind;
+use winnow::error::ParseError as _;
+use winnow::multi::{fold_many0, many0, separated0, separated1};
+use winnow::sequence::{preceded, terminated};
+use winnow::Parser;
```

Comment:
We use nightly to format our code, I guess that's why it isn't formatted as expected: `cargo +nightly fmt`